scipy/special/special/cephes/ndtr.h

```diff
@@ -0,0 +1,275 @@
+ * arithmetic   domain     # trials      peak         rms
+ *    IEEE      0,1         30000       3.7e-16     1.0e-16
+ *
+ */
+/*							erfc.c
+ *
+ *	Complementary error function
+ *
+ *
+ *
+ * SYNOPSIS:
+ *
+ * double x, y, erfc();
+ *
+ * y = erfc( x );
+ *
+ *
+ *
+ * DESCRIPTION:
+ *
+ *
+ *  1 - erf(x) =
+ *
+ *                           inf.
+ *                             -
+ *                  2         | |          2
+ *   erfc(x)  =  --------     |    exp( - t  ) dt
+ *               sqrt(pi)   | |
+ *                           -
+ *                            x
+ *
+ *
+ * For small x, erfc(x) = 1 - erf(x); otherwise rational
+ * approximations are computed.
+ *
+ *
+ *
+ * ACCURACY:
+ *
+ *                      Relative error:
+ * arithmetic   domain     # trials      peak         rms
+ *    IEEE      0,26.6417   30000       5.7e-14     1.5e-14
+ */
+
+/*
+ * Cephes Math Library Release 2.2:  June, 1992
+ * Copyright 1984, 1987, 1988, 1992 by Stephen L. Moshier
+ * Direct inquiries to 30 Frost Street, Cambridge, MA 02140
+ */
+#pragma once
+
+#include "../config.h"
+
+#include "const.h"
+#include "polevl.h"
+
+namespace special {
+namespace cephes {
+
+    namespace detail {
+
+        constexpr double ndtr_P[] = {2.46196981473530512524E-10, 5.64189564831068821977E-1, 7.46321056442269912687E0,
+                                     4.86371970985681366614E1,   1.96520832956077098242E2,  5.26445194995477358631E2,
+                                     9.34528527171957607540E2,   1.02755188689515710272E3,  5.57535335369399327526E2};
+
+        constexpr double ndtr_Q[] = {
+            /* 1.00000000000000000000E0, */
+            1.32281951154744992508E1, 8.67072140885989742329E1, 3.54937778887819891062E2, 9.75708501743205489753E2,
+            1.82390916687909736289E3, 2.24633760818710981792E3, 1.65666309194161350182E3, 5.57535340817727675546E2};
+
+        constexpr double ndtr_R[] = {5.64189583547755073984E-1, 1.27536670759978104416E0, 5.01905042251180477414E0,
+                                     6.16021097993053585195E0,  7.40974269950448939160E0, 2.97886665372100240670E0};
+
+        constexpr double ndtr_S[] = {
+            /* 1.00000000000000000000E0, */
+            2.26052863220117276590E0, 9.39603524938001434673E0, 1.20489539808096656605E1,
+            1.70814450747565897222E1, 9.60896809063285878198E0, 3.36907645100081516050E0};
+
+        constexpr double ndtr_T[] = {9.60497373987051638749E0, 9.00260197203842689217E1, 2.23200534594684319226E3,
+                                     7.00332514112805075473E3, 5.55923013010394962768E4};
+
+        constexpr double ndtr_U[] = {
+            /* 1.00000000000000000000E0, */
+            3.35617141647503099647E1, 5.21357949780152679795E2, 4.59432382970980127987E3, 2.26290000613890934246E4,
+            4.92673942608635921086E4};
+
+        constexpr double ndtri_UTHRESH = 37.519379347;
+
+    } // namespace detail
+
+    SPECFUN_HOST_DEVICE inline double erf(double x);
```

Comment:
I'd try to avoid prototype declarations, and would just move the actual `erf` body to be above `erfc`.

Reply:
They're mutual recursive though.

Reply:
Didn't catch that, makes sense!